pkg/generator/generator.go

```diff
@@ -0,0 +1,22 @@
+package generator
+
+import (
+	"context"
+
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/internal/events"
+)
+
+type ISOGenerator interface {
```

Comment:
Why not define a mock here? then you will not need to have a job mock in inventory_test?